mmv1/products/vertexai/Models.yaml

```diff
@@ -0,0 +1,425 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: Models
+base_url: projects/{{project}}/locations/{{region}}/models
+create_url: 'projects/{{project}}/locations/{{region}}/modelsPRE_CREATE_REPLACE_ME'
+delete_url: '{{name}}'
+self_link: '{{name}}'
+
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/vertex-ai/docs'
+  api: 'https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.models'
+description: |-
+  A trained machine learning Model.
+id_format: '{{name}}'
+update_verb: 'PATCH'
+exclude_import: true
+async:
+  actions: ['create', 'delete', 'update']
+  type: 'OpAsync'
+  operation:
+    base_url: '{{op_id}}'
+    path: 'name'
+    wait_ms: 1000
+  result:
+    path: 'response'
+    resource_inside_response: true
+  error:
+    path: 'error'
+    message: 'message'
+custom_code:
+  encoder: templates/terraform/encoders/vertex_ai_models.tmpl
+  decoder: templates/terraform/decoders/vertex_ai_models.tmpl
+  pre_create: templates/terraform/pre_create/vertex_ai_models.tmpl
+examples:
+  - name: 'vertex_ai_model_source_basic'
+    exclude_import_test: true
+    primary_resource_id: 'model'
+    vars:
+      model_id: 'test-model'
+    test_env_vars:
+      project_name: 'PROJECT_NAME'
```

Comment:
Please check how this test is being generated- currently the config for the test includes:

```
resource "google_vertex_ai_models" "model" {
  display_name = ""
  project = ""
```

Reply:
you're referring to the generated test? I checked and getting the following generation:
```hcl
func testAccVertexAIModels_vertexAiModelSourceBasicExample(context map[string]interface{}) string {
	return acctest.Nprintf(`
resource "google_vertex_ai_models" "model" {
  model_id = "tf-test-test-model%{random_suffix}"
  project = "%{project_name}"
  source_model = "projects/%{project_name}/locations/us-central1/models/4469245519628878651"

  region       = "us-central1"
}
`, context)
}
```

Reply:
Ah, I think I had a stale version of your branch pulled locally, sorry!